chainercv/datasets/coco/coco_instance_segmentation_dataset.py

```diff
@@ -56,6 +48,10 @@ class COCOInstanceSegmentationDataset(GetterDataset):
             under :obj:`$CHAINER_DATASET_ROOT/pfnet/chainercv/coco`.
         split ({'train', 'val', 'minival', 'valminusminival'}): Select
             a split of the dataset.
+        year ({'2014', '2017'}): Use a dataset prepared for a challenge
+            held in :obj:`year`.
```

Comment:
The competition that used COCO 2014 was held in 2015. http://cocodataset.org/#detection-2015

Reply:
thanks